### PR TITLE
Adds Missing Disease Symptoms Take 2

### PR DESCRIPTION
**Pull request by Alexshreds**
Fix - #7454

Hyphema and ocular restoration were missing.

This PR comes with less git boogaloo.

🆑 Alexshreds
bugfix: Hyphema and Ocular Restoration are now possible to evolve from viruses
/ 🆑

### CODE REVIEW

File: code/datums/diseases/advance/symptoms/vision.dm [hunk 1]
```diff
@@ -29,22 +29,24 @@ Bonus
...
 		var/mob/living/M = A.affected_mob
+		var/obj/item/organ/internal/eyes/eyes = M.get_int_organ(/obj/item/organ/internal/eyes)
 		switch(A.stage)
 			if(1, 2)
 				to_chat(M, "<span class='warning'>Your eyes itch.</span>")
 			if(3, 4)
 				to_chat(M, "<span class='warning'><b>Your eyes burn!</b></span>")
-				M.blur_eyes(10)
-				M.adjust_eye_damage(1)
+				M.SetEyeBlurry(20)
```
Comment (Fox-McCloud) [inline code/datums/diseases/advance/symptoms/vision.dm]: `M.EyeBlurry(10)`

File: code/datums/diseases/advance/symptoms/vision.dm [hunk 2]
```diff
@@ -29,22 +29,24 @@ Bonus
...
+				eyes.take_damage(1)
 			else
 				to_chat(M, "<span class='userdanger'>Your eyes burn horrificly!</span>")
-				M.blur_eyes(20)
-				M.adjust_eye_damage(5)
-				if(M.eye_damage >= 10)
-					M.become_nearsighted()
-					if(prob(M.eye_damage - 10 + 1))
-						if(M.become_blind())
+				M.SetEyeBlurry(30)
```
Comment (Fox-McCloud) [inline code/datums/diseases/advance/symptoms/vision.dm]: `M.EyeBlurry(20)`

File: code/datums/diseases/advance/symptoms/vision.dm [hunk 3]
```diff
@@ -29,22 +29,24 @@ Bonus
...
+				eyes.take_damage(5)
+				if(eyes.damage >= 10)
+					M.BecomeNearsighted()
+					if(prob(eyes.damage - 10 + 1))
+						if(M.BecomeBlind())
 							to_chat(M, "<span class='userdanger'>You go blind!</span>")
+	return
```
Comment (Fox-McCloud) [inline code/datums/diseases/advance/symptoms/vision.dm]: Trailing return.

**Comment by Fox-McCloud** [comment]
This will runtime if something doesn't have eyes; likewise, it's typecasted for `living`, despite only `carbon` and up being capable of even having eye organs.

File: code/datums/diseases/advance/symptoms/vision.dm (hunk 3 above)
Comment (Alexshreds) [inline code/datums/diseases/advance/symptoms/vision.dm]: They all have returns like this

**Comment by Alexshreds** [comment]
I changed the blur amount because it was super wimpy and didn't last more than a couple seconds.

**Pull request merged**